### ISSUE TITLE
Document deployment guide

### ISSUE THREAD

**Issue by JAORMX** [latchset/custodia#150]
Besides the minimal example referenced in #36, having a small deployment guide that describes how to deploy with some common scenarios would be very beneficial.

**Comment by simo5** [comment]
Do you have an example in mind I can look out at ?
I want to take on this soon.

**Comment by JAORMX** [comment]
So it could be something similar to https://docs.openstack.org/ocata/install-guide-rdo/keystone.html where a basic installation workflow is followed, as well as operations that deployers would usually do and how to verify them.

And if possible some "advanced features" that people might actually need; such as security hardening and such.

**Comment by simo5** [comment]
Thanks, will take a look and start implementing what I can.

**Comment by tiran** [comment]
Deployment should be either via package manager or with pip in a virtual env. Installation with ```sudo pip install custodia``` in global site-packages should be discourage. I don't want to support it because it tends to break either custodia, the system or both.

**Comment by JAORMX** [comment]
@tiran I agree that `sudo pip install custodia` shouldn't be encouraged. And that's the aim of having a deployment guide. To tell folks right ways of doing things such as installing and configuring the service.
